apps/io-services-cms-webapp/src/reviewer/request-review-legacy-handler.ts

```diff
@@ -0,0 +1,116 @@
+import { Context } from "@azure/functions";
```

Comment:
I'd add a top level comment with the purpose of this module

Reply:
74b836e